### PR TITLE
Changed -dev to -sha in Mesos change log links.

### PR DESCRIPTION
**Pull request by anithamuthu79**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@orsenthil @vinodkone this PR is ready for review.

File: pages/1.10/release-notes/1.10.6/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ DC/OS 1.10.6 was released on April  24, 2018.
 [button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
 
 DC/OS 1.10.6 includes the following:
-- Apache Mesos 1.4.2-dev[change log](https://github.com/apache/mesos/blob/24d3886/CHANGELOG)
+- Apache Mesos 1.4.2-24d388[change log](https://github.com/apache/mesos/blob/24d3886/CHANGELOG)
```
Comment (orsenthil) [inline pages/1.10/release-notes/1.10.6/index.md]: Hi @anithamuthu79  - I did a mistake. Let's go with 7 characters.  `1.4.2-24d3886` here and through out.

File: pages/1.10/release-notes/1.10.5/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ DC/OS 1.10.5 was released on February 22, 2018.
 # Notable Changes in DC/OS 1.10.5
 
 - Updated to Marathon 1.5.6 ([changelog](https://github.com/mesosphere/marathon/releases/tag/v1.5.6)).
-- Updated to Mesos 1.4.2-dev ([changelog](https://github.com/apache/mesos/blob/92d988ce0192ef8cc8bc75575c3a7d18c30ab7ea/CHANGELOG)).
+- Updated to Mesos 1.4.2-92d988c ([changelog](https://github.com/apache/mesos/blob/92d988ce0192ef8cc8bc75575c3a7d18c30ab7ea/CHANGELOG)).
```
Comment (vinodkone) [inline pages/1.10/release-notes/1.10.5/index.md]: let's use the short sha in the URL too for consistency with how RNs for other releases look below.

File: pages/1.10/release-notes/1.10.8/index.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ DC/OS 1.10.8 was released on July 19, 2018.
 [button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
 
 DC/OS 1.10.8 includes the following:
-- Apache Mesos 1.4.2-dev [change log](https://github.com/apache/mesos/blob/5f6d4cdc79e1b28bc2d157d4b5b4aaf941a26d91/CHANGELOG).
+- Apache Mesos 1.4.2-5f6d4cd [change log](https://github.com/apache/mesos/blob/5f6d4cdc79e1b28bc2d157d4b5b4aaf941a26d91/CHANGELOG).
```
Comment (vinodkone) [inline pages/1.10/release-notes/1.10.8/index.md]: short sha.

File: pages/1.11/release-notes/1.11.4/index.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ DC/OS 1.11.4 was released on July 26, 2018.
 
 DC/OS 1.11.4 includes the following:
 
-- Apache Mesos 1.5.2-dev [change log](https://github.com/apache/mesos/blob/2440c7310627c835fe828aef6b28b8d072e7371c/CHANGELOG).
+- Apache Mesos 1.5.2-2440c73 [change log](https://github.com/apache/mesos/blob/2440c7310627c835fe828aef6b28b8d072e7371c/CHANGELOG).
```
Comment (vinodkone) [inline pages/1.11/release-notes/1.11.4/index.md]: short sha.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ DC/OS 1.11.3 was released on June 26, 2018.
 
 DC/OS 1.11.3 includes the following:
 
-- Apache Mesos 1.5.2-dev [change log](https://github.com/apache/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
+- Apache Mesos 1.5.2-197c910 [change log](https://github.com/apache/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
```
Comment (vinodkone) [inline pages/1.11/release-notes/1.11.3/index.md]: short sha.

**Comment by anithamuthu79** [comment]
@vinodkone @orsenthil I've replaced long SHA's with short SHA's. Please review and share your feedback. TY.

**Comment by anithamuthu79** [comment]
@vinodkone could you review this PR?

**Comment by anithamuthu79** [comment]
@vinodkone as per your suggestion, I've posted this PR on #core channel for review. Gilbert has reviewed and approved this fix. I'm merging and closing this PR. Ty.

**Pull request merged**